lambdas/appointment_email.py

```diff
@@ -360,10 +198,12 @@ def send_cal_email(supp, supp_email, student, student_email, appt_time, duration
     # eml_atch = MIMEBase('text/plain','')
     # encoders.encode_base64(eml_atch)
     # eml_atch.add_header('Content-Transfer-Encoding', "")
-
+    
     msgAlternative.attach(part_email)
+    msg.attach(msg_content)
     msgAlternative.attach(part_cal)
-
+ 
+    
```

Comment:
Please keep your coding style consistent throughout. Separation of code by just one line is fine. No need for extra lines.